### PR TITLE
docs: 📚 add bug-hint to Storybook 

### LINKED ISSUE #463: docs: 📚 add bug-hint to Storybook
## Description
As a user of Storybook, I would like to quickly know how to handle a bug I just found while going through the library. Therefore I suggest to add a line of text to the top-bar in storybook that says:
"Did you find a bug? Please check the [Know Browser Issues](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-known-browser-issues--docs) before you report an issue on [Jira](https://jira.apps.union-investment.de/browse/PSOLIDDS) or [GitHub](https://github.com/solid-design-system/solid/issues)."
## Where to put the documentation
Storybook

## Technical Information
Storybook offers to write custom addons to implement something like this: https://storybook.js.org/docs/react/addons/writing-addons

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

### LINKED ISSUE #588: fix: 🤔 Typo in Theme Generator
## Current behavior
Typo -> Soid Theme Generator
image

## Expected behavior
Fix Typo -> Solid Theme Generator

## DoR
- [x] Item has business value
- [x] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

### PR DESCRIPTION
**Pull request by Vahid1919**
## Description:
Added a bug-hint as a toolbar add-on. Closes #463.
Closes #588.

## Definition of Reviewable:
*PR notes: Irrelevant elements should be removed.*
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

File: packages/components/.storybook/addons/constants.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export const ADDON_ID = 'solid/theme-generator';
```
Comment (mariohamann) [inline packages/components/.storybook/addons/constants.ts:1]: You addon should have an own ID. I don't think that you should reuse this file.

**Comment by Vahid1919** [comment]
@mariohamann Right now on smaller screens the toolbar becomes scrollable. 
I am not sure if this is ideal. What do you think?
image

File: packages/components/.storybook/addons/bug-hint/Tool.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        color: 'black',
+        display: 'flex',
+        alignItems: 'center',
+        fontSize: '14px',
+        userSelect: 'none'
+      }}
+    >
+      <span>
+        Did you find a bug? Please check the{' '}
+        <a href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/docs-general-known-browser-issues--docs">
```
Comment (mariohamann) [inline packages/components/.storybook/addons/bug-hint/Tool.tsx:16]: Please setup some styling for the link, that the color fits to the main color we have in Storybook.

File: packages/components/.storybook/addons/bug-hint/Tool.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          Known Browser Issues
+        </a>{' '}
+        before you report an issue on <a href="https://jira.apps.union-investment.de/browse/PSOLIDDS"> Jira </a> or{' '}
```
Comment (mariohamann) [inline packages/components/.storybook/addons/bug-hint/Tool.tsx:19]: Please check the spaces inside the links, they are not perfect leading to strange underlines.

File: packages/components/.storybook/addons/theme-generator/Panel.tsx
Comment (mariohamann) [inline packages/components/.storybook/addons/theme-generator/Panel.tsx:1]: Could you please do me a favor and rename `Soid Theme Generator` to `Solid Theme Generator` in this file?

File: packages/components/.storybook/addons/bug-hint/Tool.tsx (hunk 2 above)
Comment (Vahid1919) [reply packages/components/.storybook/addons/bug-hint/Tool.tsx]: I added the primary color for the anchor. But it seems that adding pseudo-classes (eg: :visited, :active) are not super straightforward in React. @mariohamann Should I invest the time to try to implement them too, or is the current update enough?

Comment (mariohamann) [reply packages/components/.storybook/addons/bug-hint/Tool.tsx]: No. It's enough.

**Review by karlbaumhauer — APPROVED** [review]
LGTM

**Review by azraefendic — APPROVED** [review]

**Review by abudd1094 — APPROVED** [review]

File: packages/components/.storybook/addons/manager.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { addons, types } from '@storybook/manager-api';
+import { ADDON_ID_BUG_HINT, TOOL_ID } from './bug-hint/constants';
```
Comment (mariohamann) [inline packages/components/.storybook/addons/manager.ts:2]: Please call both exports just ADDON_ID and change the import via `as` https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

**Comment by yoezlem** [comment]
What should someone do if they find a bug but do not have access to our Jira board, especially if they are outside of Union Investment?

**Comment by karlbaumhauer** [comment]
> What should someone do if they find a bug but do not have access to our Jira board, especially if they are outside of Union Investment? @karlbaumhauer @mariohamann @coraliefeil

they can use GitHub OR Jira... so from my perspective, these are enough ways to provide a bug... But for the future, we could set up an support mail for those, who dont want to create / have a GitHub account. But as people from outside are most probably developers (GitHub is the only public area of our project), I would like to see the need before we open up another channel which needs to be taken care of...

**Review by mariohamann — APPROVED** [review]

**Review by azraefendic — APPROVED** [review]

**Pull request merged**